apps/maestro/src/server/routers/interchainToken/recordRemoteTokensDeployment.ts

```diff
@@ -72,22 +69,9 @@ export const recordRemoteTokensDeployment = protectedProcedure
           }),
         ]);
 
-        const tokenManagerClient = ctx.contracts.createTokenManagerClient(
-          configs.wagmi,
-          tokenManagerAddress
-        );
-
-        const tokenManagerTypeCode = await tokenManagerClient.reads
-          .implementationType()
-          .catch(() => null);
-
-        const tokenManagerType = Maybe.of(tokenManagerTypeCode).mapOrNull(
-          getTokenManagerTypeFromBigInt
-        );
-
         return {
           tokenManagerAddress,
-          tokenManagerType,
+          tokenManagerType: "mint_burn" as const,
```

Comment:
it's "safe" to do this for now but once we expand to other token manager types for remote tokens, this will need to change. similar comment to above, can it programmatically be set based on the enum value? https://github.com/axelarnetwork/axelarjs/blob/main/apps/maestro/src/lib/drizzle/schema/common.ts#L47